### DISCUSSION TITLE
Multiple different claims for keycloak OAuth2 provider

### DISCUSSION THREAD

**Discussion by rd-dreich** [provectus/kafka-ui#4139] [General]
Hi folks,
we're currently checking out Kafka-UI and so far it looks fine. However, we're using keycloak as an OAuth2 provider and want to use both group and user claims for RBAC. Is there a way to set this up? Because it looks like you could only pick one claim in the roles-field setting when using the generic version.
Thanks

**Comment by Haarolean** [comment]
Hey, currently it's not possible. Though, sounds like a design issue, if only the field would be settable at the subject's level in RBAC configuration rather than in OAuth configuration 🤔 I'll convert this into an issue

**Comment by Haarolean** [comment]
#4223
